### DISCUSSION TITLE
Daily 26/09

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [grupo-ispc-2023-2-0/workflix-app#108] [Daily]
Con respecto al proyecto:

- ¿Qué hice ayer?
- ¿Qué hice o haré hoy?
- Impedimentos o dificultades

**Comment by NataliaAlvarezIspc** [comment]
Ayer termine la vista de la activity de login, me falta conectarla con registro. También cree la activity de registro para empezar a trabajar y subí todo al repo.
Hoy voy a realizar la activity de registro. Por la tarde voy a participar de la clase sincrónica.

**Comment by Valentino399** [comment]
Antes de ayer estuve practicando en Android studio, ayer estuve viendo videos del programa y hoy me pondré con el proyecto.

**Comment by BraianTroncoso** [comment]
Hice las tareas que me habia asignado. Terminé con el menú de navegación, el home, y el formulario de contacto con sus respectivos activitys, fragments, vectores y xml. Dejé todo listo para vincular a todo lo demás ya que el menú es el esqueleto que conecta todo. 
Agregue las imágenes a drawable.
También solucioné un bug que al parecer esta version de java con sdk los switch generan conflictos y hay error en tiempo de compilación lo cual desaparece si usamos if anidados.

**Comment by MarielaGomez** [comment]
Estuvimos con mis compañeros intercambiando formas de implementar los distintos activitys que nos correspondían con respecto al perfil, han habido complicaciones para poder emular, errores de aplicativos, falta de recursos y conocimientos para el manejo del diseño front, a la vez que ibamos leyendo materiales y buscando por internet resoluciones.
